package.json

```diff
@@ -70,6 +66,9 @@
     "build-proto-roundtrip": "cd proto/scripts && PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python json_roundtrip_via_proto.py"
   },
   "devDependencies": {
+    "@firebase/app-types": "0.3.1",
```

Comment:
😬

Reply:
> 😬

yeah, this is only for the types. We use so little of it, maybe we should make our own types version locally